packages/core/src/components/bal-form/bal-form.tsx

```diff
@@ -17,11 +19,18 @@ export class Form implements ComponentInterface {
    */
   @Prop() novalidate = false
 
+  /**
+   * The css class for the inner form element
+   */
+  @Prop() formClass = ''
```

Comment:
Is this just set in consuming applications? Like Motorvehicle? Would it make sense to add an example?

Reply:
the problem is that the inner element can break styling of an application and with that prop you can style the inner element with css class helpers